src/components/FAQ.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import FAQItem from "./FAQItem";
+import faqData from "./faqData";
+import starIcon from "../assets/images/icon-star.svg";
+
+export default function FAQ() {
+  return (
+    <div className="faq-app">
+      <div className="background-container">
+        <div className="background-pattern"></div>
+        <div className="background-color"></div>
+      </div>
+
+      <main className="faq-container">
+        <div className="faq-header">
+          <img src={starIcon} alt="Star icon" className="star-icon" />
+          <h1>FAQs</h1>
+        </div>
+
+        <div className="faq-content">
+          {faqData.map((Item, index) => (
+            <FAQItem key={index} question={Item.question} answer={Item.answer} />
```

Comment:
Do not use `index` as your key. use something unique instead like the` question` or the `answer` since the values are unique or you can add an `id` to your faq data object and use that for the key instead.
You will learn more about React keys next week as we deep dive into React.